cas-server-core-authentication/src/main/java/org/apereo/cas/web/flow/AuthenticationExceptionHandler.java

```diff
@@ -118,9 +130,9 @@ public void setErrors(final List<Class<? extends Exception>> errors) {
      * @return true if any custom errors have been added, false otherwise.
      */
     final boolean containsCustomErrors() {
-        return !(DEFAULT_ERROR_LIST.size() == this.errors.size()) && (this.errors.containsAll(DEFAULT_ERROR_LIST));
+        return !(DEFAULT_ERROR_LIST.size() == this.errors.size()) && this.errors.containsAll(DEFAULT_ERROR_LIST);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use opposite operator instead of the logic complement operator.](https://www.codacy.com/app/mmoayyed/cas/file/2770326140/issues/source?bid=3256115&fileBranchId=3357337#l133)